api/services/account_service.py

```diff
@@ -672,7 +672,7 @@ def get_tenant_count() -> int:
         return db.session.query(func.count(Tenant.id)).scalar()
 
     @staticmethod
-    def check_member_permission(tenant: Tenant, operator: Account, member: Account, action: str) -> None:
+    def check_member_permission(tenant: Tenant, operator: Account, member: Account | None, action: str) -> None:
```

Comment:
This seems the same with operator, why it's None?

Reply:
https://github.com/langgenius/dify/blob/f458580dee84066b0bbd1b32eb3359b90de2125d/api/services/account_service.py#L837
Here a None is passed in directly.